platform/master/R/master.R

```diff
@@ -443,3 +443,37 @@ check_dr_version_compatibility<-function(){
   return (TRUE)
 }
 
+dload <- function(x, trace = FALSE){
+
+  number_of_executors <- sum(distributedR_status()$Inst)
+  y <- lapply(x, dloadHelper, trace=trace, num_of_Exec = number_of_executors)
+
+}
+
+dloadHelper <- function(x, trace = FALSE, num_of_Exec) {
```

Comment:
Pls make helper functions hidden and possible to change the name to be more specific to loading libraries?